tests/test_integration.py

```diff
@@ -101,6 +101,14 @@ def test_run_gams_api():
     assert np.isclose(obs, exp)
 
 
+def scenario_list(mp):
+    return mp.scenario_list(default=False)[['model', 'scenario']]
+
+
+def assert_multi_db(mp1, mp2):
```

Comment:
Can this somehow also check that `mp1` and `mp2` are distinct?

Reply:
The thing is we should (in general) check whether both platform instances pointing to different databases (which might be tricky)

Reply:
Okay, maybe that's a separate issue: either the Java class could expose this information, or it could be stored as properties of the `ixmp.Platform` by the constructor. No need to fix here.

Reply:
It might be still cases when we have different JDBC URLs for same DB instance.

Reply:
by construction of this test, the `mp` instances will be different